### PR TITLE
chore: add itest to convert document to pdf

### PR DESCRIPTION
**Pull request by edgarvonk**
Add integration test to convert an uploaded text document to pdf using the Office Converter Docker container.

Solves PZ-4226

### CODE REVIEW

File: src/itest/kotlin/nl/lifely/zac/itest/EnkelvoudigInformatieObjectRestServiceTest.kt [hunk 1]
```diff
@@ -235,6 +245,130 @@ class EnkelvoudigInformatieObjectRestServiceTest : BehaviorSpec({
...
+                    )
+                    shouldContainJsonKeyValue(
+                        "informatieobjectTypeOmschrijving",
+                        INFORMATIE_OBJECT_TYPE_BIJLAGE_OMSCHRIJVING
+                    )
+                    shouldContainJsonKey("informatieobjectTypeUUID")
+                    shouldContainJsonKey("identificatie")
+                    shouldContainJsonKeyValue("bestandsnaam", TEST_TXT_CONVERTED_TO_PDF_FILE_NAME)
+                    shouldContainJsonKeyValue("formaat", PDF_MIME_TYPE)
+                }
```
Comment (hsiliev) [inline src/itest/kotlin/nl/lifely/zac/itest/EnkelvoudigInformatieObjectRestServiceTest.kt:369]: Would this be better with `shouldEqualJson` ?

**Review by hsiliev — APPROVED** [review]

File: src/itest/kotlin/nl/lifely/zac/itest/EnkelvoudigInformatieObjectRestServiceTest.kt (hunk 1 above)
Comment (edgarvonk) [reply src/itest/kotlin/nl/lifely/zac/itest/EnkelvoudigInformatieObjectRestServiceTest.kt:369]: yep. I just copied it from another block but you're right I guess. although we probably skip a few vars here..

Comment (hsiliev) [reply src/itest/kotlin/nl/lifely/zac/itest/EnkelvoudigInformatieObjectRestServiceTest.kt:369]: We can probably use the shortcuts we already have to ignore missing key-value pairs

Comment (edgarvonk) [reply src/itest/kotlin/nl/lifely/zac/itest/EnkelvoudigInformatieObjectRestServiceTest.kt:396]: done :-)

**Review by hsiliev — APPROVED** [review]

**Pull request merged**